Eugene/MovieSearch/my-app/src/components/MainPage/main/Main.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import "./Main.css";
+import IMovie from "../../../interface/IMovie/IMovie";
+import Movie from "../movie/Movie";
+
+interface IMainProps {
+  movies: IMovie[];
+}
+
+class Main extends React.Component<IMainProps,{}> {
```

Comment:
doesn't seem like a right name for such component. It renders movies list, so call it like this. also "main" tag should wrap all main content on the page. I would say that search form also should be included and movie description for sure. So, please remove "main" tag from here. https://developer.mozilla.org/ru/docs/Web/HTML/Element/main